### DISCUSSION TITLE
Add new device -- Basic instructions / explanation?

### DISCUSSION THREAD

**Discussion by jonathancross** [cryptoadvance/specter-desktop#950] [Specter UI/UX]
I am trying to add an "other" device via xpub (technically zpub) for use in a multisig wallet.  I find the UX very confusing and have no idea what is expected, eg:
1. Why doesn't `#0 Multisig Sig (Segwit)` line item simply let me paste in the zpub there, but is fixed as `-`?
2. When I click on "Paste xpub", I get a brand new line item appended with no ability to edit the derivation path.
3. Why are items like "#0 Multisig Sig (Segwit)" editable?  Are the xpubs supposed to go in there?

I looked through the documentation but didn't see an explanation / tutorial on what is expected here.

UX needs work, but I am too confused to make suggestions at this point.

**Comment by ben-kaufman** [comment]
Hi, so we intentionally don't allow manual editing of the xpubs from the table because it is very error-prone. Also pasting an xpub is very error prone which is why we enable it only in the "Other" type.

When pasting, the table will populate the xpub in an existing column if the derivation match or can be auto detected - which is possible with `zpub` but not with `Zpub`, since there is [no definded derivation for Zpub](https://github.com/satoshilabs/slips/blob/master/slip-0132.md). So it would have worked with `zpub`, but with `Zpub` or `xpub, where we can't know the derivation, the derivation needs to be included in the pasted Zpub like this: `[fingerprint/derivation]Zpub`.

**Reply by jonathancross** [reply]
Thanks for the reply.  I am not clear though.  It sounds like you are suggesting users should concatenate the derivation path with the `Zpub` before pasting?  Can you provide an example?

You are saying that pasting a `zpub` _should work fine_ and auto-populate? (didn't in my case, but I can experiment more)

Referring to the line item numbers I used above, I'm not clear about what you are saying for number 3.

Thanks.

**Reply by ben-kaufman** [reply]
Hi, for #3, no these are just for labeling the xpubs so you'll know the purpose of each.

For Zpub format example, here's one with Vpub (testnet equivalent)
`[6ea15da6/48h/1h/0h/2h]Vpub5mvDKGjRsucYGM7PWahGKJKkC3um3oJMqCYDf6SzdkPp8yES65bLfvxhE1bCZsqWobZKpcdTk3niqKR3f6T4B2zJDSyDdes3TyRM17vXYQs`

**Comment by hollisp01** [comment]
Hi, newbie here.   I am have my Lightning Node synched and running on Raspberry Pi 4.   When I run Specter App within umbrel running on Laptop I get a test that it is connected to the core successfully.  When I attempt to add a Trezor 1 wallet automatically it via USB does not detect the wallet.  I have plugged the wallet into the Raspberi Pi USB port.  I have the Umbrel and my Laptop connected via ethernet to the same switch router.
